File: C Programming/Competitive Programming/Coding Challenges 8/program4.c
/*
4. Write a program which accept temperature in Fahrenheit and convert it into celsius. (1 celsius = (Fahrenheit -32) * (5/9))
Input : 10
Output :  -12.222222 (10 - 32) * (5.0/9.0)
Input : 34
Output :  1.111111 (34 - 32) * (5.0/9.0)

*/

#include<stdio.h>

double FhtoCs (float fTemp)
{
    double dCelsius = 0.0;
    dCelsius = (fTemp-32)*(5.0/9.0);
    return dCelsius;
}

int main()
{
    float fValue = 0.0f;
    double dRet = 0.0;

    printf("Enter temperature in Fahrenheit: ");
    scanf("%f",&fValue);

    dRet = FhtoCs(fValue);

    printf("Temperature in Celsius is : %lf",dRet);
    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 14/program4.java
/*
4. Write a program which accept temperature in Fahrenheit and convert it into celsius.
(1 celsius = (Fahrenheit -32) * (5/9))
Input : 10
Output : -12.222222
Input : 34
Output : 1.111111
*/

import java.util.Scanner;

class program4
{
    static double FhtoCs(float fTemp)
    {
        double dDifference = fTemp - 32;
        double dCelsius = (dDifference * 5) / 9;

        return dCelsius;
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        float fValue = 0.0f;
        double dRet = 0.0;

        System.out.print("Enter temperature in Fahrenheit: ");
        fValue = sobj.nextFloat();

        dRet = FhtoCs(fValue);

        System.out.print("Temperature in Celsius is : " + dRet);

        sobj.close();
    }
}